unix/Makefile

```diff
@@ -108,6 +114,19 @@ SRC_MOD += modjni.c
 endif
 
 # source files
+ifeq ($(TARGET), freedos)
+# no moduselect for freedos
+SRC_C = \
+	main.c \
+	gccollect.c \
+	unix_mphal.c \
+	input.c \
+	file.c \
+	modos.c \
+	alloc.c \
+	coverage.c \
+	$(SRC_MOD)
+else
```

Comment:
@pohmelie : you asked how to do it on makefile level, @dhylands told you, that doesn't imply how it should be done in this case ;-)

 > All other configurable modules are configured in mpconfigport.mk.

Well, not quite. mpconfigport.mk configures what can't be configured via mpconfigport.h (and that's specifically stuff which requires linking with extra libraries). Well, there're some weird exceptions (which probably should be cleaned up).